monitor/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -55,6 +55,8 @@
 <sec:authorize access="isAuthenticated()">
 	<p>You are logged in.</p>
 </sec:authorize>
+<p>Deployed Version: <c:set var="version" value="<%= this.getClass().getPackage().getImplementationVersion() %>" />
```

Comment:
I doubt that this provides the correct version number. This should be done through properties and the pom. Check out Virtual Spaces or Citesphere for examples.